File: Folder C++/Kode/Person.cpp
#ifndef PERSON_CPP
#define PERSON_CPP

#include <iostream>
#include <string>
using namespace std;

class Person {
private:
    string nama;
    string alamat;
    int usia;
    string gender;

public:
    Person() {}
    Person(string nama, string alamat, int usia, string gender)
        : nama(nama), alamat(alamat), usia(usia), gender(gender) {}

    void setNama(string nama) { this->nama = nama; }
    string getNama() const { return nama; }

    void setAlamat(string alamat) { this->alamat = alamat; }
    string getAlamat() const { return alamat; }

    void setUsia(int usia) { this->usia = usia; }
    int getUsia() const { return usia; }

    void setGender(string gender) { this->gender = gender; }
    string getGender() const { return gender; }

    Person& operator=(const Person& other) {
        if (this != &other) {
            nama = other.nama;
            alamat = other.alamat;
            usia = other.usia;
            gender = other.gender;
        }
        return *this;
    }
};

#endif

File: Folder Python/Kode/Person.py
# Person.py
class Person:
    def __init__(self, nama="", alamat="", usia=0, gender=""):
        self.__nama = nama
        self.__alamat = alamat
        self.__usia = usia
        self.__gender = gender

    def get_nama(self):
        return self.__nama

    def set_nama(self, nama):
        self.__nama = nama

    def get_alamat(self):
        return self.__alamat

    def set_alamat(self, alamat):
        self.__alamat = alamat

    def get_usia(self):
        return self.__usia

    def set_usia(self, usia):
        self.__usia = usia

    def get_gender(self):
        return self.__gender

    def set_gender(self, gender):
        self.__gender = gender